boilerplate/index.js.ejs

```diff
@@ -7,5 +7,9 @@
 // side effect of breaking other tooling like mobile-center and react-native-rename.
 //
 // It's easier just to leave it here.
-
+<% if (props.useExpo) { %>
+import App from "./app/app.tsx"
+export default App
```

Comment:
Expo uses the customized AppRegistry called `registerRootComponent` internally, we need to explicitly export default.

Reply:
Let's move the other app register code from `./app/app.tsx` here too, so we can `import App from "./app/app.tsx"` for both types, and then in the `if/else` we can either export default (expo) or register the app (RN CLI).